apps/web/src/components/Follow/FollowListUsers.tsx

```diff
@@ -51,12 +68,32 @@ export default function FollowListUsers({
   return (
     <StyledList>
       {formattedUsersBio.map((user) => (
-        <StyledListItem key={user.dbid} href={`/${user.username}`} onClick={handleClick}>
-          <HStack gap={4} align="center" inline>
-            <ProfilePicture userRef={user} size="sm" />
-            <TitleS>{user.username}</TitleS>
+        <StyledListItem
+          key={user.dbid}
+          href={`/${user.username}`}
+          onClick={handleClick}
+          isMobile={isMobile}
+        >
+          <HStack gap={8}>
+            <ProfilePicture userRef={user} size="md" />
+            {user.bio ? (
```

Comment:
instead of a ternary here to return either username + bio or username, should be able to do something like this (pseudocode example)

<div>
<title>username</title>
bio && <markdown bio/>
</div>

can also peep similar example in `SharedInfoListRow`